cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1587,4 +1587,10 @@
     </description>
   </property>
 
+  <!-- Dataset extensions -->
+  <property>
+    <name>dataset.extensions.dir</name>
+    <value>/opt/cdap/ext/lib</value>
+  </property>
+
```

Comment:
Did not see this until the request was merged; will add to further work on this functionality if required.